### PR TITLE
TTS finetune changes

### PR DESCRIPTION
**Pull request by mgrafu**
* Add tutorial to record a TTS dataset at home
* Update finetuning tutorial to use a subset of hifiTTS

### CODE REVIEW

File: tts-dataset-recording-at-home.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+<img width="450" alt="image" src="https://github.com/mgrafu/tutorials/assets/47233618/c08c2fbd-0f3d-46e9-aa92-f51519d183d6">
+    
+    
+8.  Select the most quiet room in your environment, and close your windows and doors.
+    
+9.  Eliminate external sources of noise, e. g. air conditioning, computer fan, kids, etc...
+
+## How to adjust microphone levels and body position before recording?
+
+1.  Set your microphone gain to ‘9 hours.’
```
Comment (virajkarandikar) [inline tts-dataset-recording-at-home.md:59]: `9 hours.` is there a typo here?

File: tts-dataset-recording-at-home.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# Guidelines to Record a TTS Dataset at Home
+
+This section describes best practices for getting the best audio quality when recording TTS data at home.
+
+## What data?
+Start by recording the [harvard sentences](http://www.cs.cmu.edu/afs/cs.cmu.edu/project/fgdata/OldFiles/Recorder.app/utterances/Type1/harvsents.txt), which should produce about 20 to 40 minutes of usable data for TTS.
+
+## What gear?
```
Comment (LynseyFabel) [inline tts-dataset-recording-at-home.md:8]:
```suggestion
## Hardware Requirements
```

File: tts-dataset-recording-at-home.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
+# Guidelines to Record a TTS Dataset at Home
+
+This section describes best practices for getting the best audio quality when recording TTS data at home.
+
+## What data?
```
[the remark this answers is not in the document]
Comment (LynseyFabel) [reply tts-dataset-recording-at-home.md:5]: Or rename to Dataset.  Trying to be consistent across all tutorials.

File: tts-dataset-recording-at-home.md (hunk 2 above)
Comment (LynseyFabel) [reply tts-dataset-recording-at-home.md:8]: Trying to be consistent across all tutorials.  We don't use the term gear.

File: tts-dataset-recording-at-home.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+Start by recording the [harvard sentences](http://www.cs.cmu.edu/afs/cs.cmu.edu/project/fgdata/OldFiles/Recorder.app/utterances/Type1/harvsents.txt), which should produce about 20 to 40 minutes of usable data for TTS.
+
+## What gear?
+
+* Use a decent cheap microphone like the [Audio-Technica AT2020USB+](https://www.amazon.com/gp/product/B00B5ZX9FM/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1) or the [blue yeti usb mic](https://www.amazon.com/Blue-Yeti-USB-Microphone-Silver/dp/B002VA464S).
```
Comment (LynseyFabel) [inline tts-dataset-recording-at-home.md:10]:
```suggestion
* Use a microphone like the [Audio-Technica AT2020USB+](https://www.amazon.com/gp/product/B00B5ZX9FM/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1) or the [Blue Yeti USB mic](https://www.amazon.com/Blue-Yeti-USB-Microphone-Silver/dp/B002VA464S).
```

File: tts-dataset-recording-at-home.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+* Get a boom filter or windscreen.
+
+## What audio recording software?
```
Comment (LynseyFabel) [inline tts-dataset-recording-at-home.md:14]: please reword so this is not a question

File: tts-dataset-recording-at-home.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+This document focuses on [Audacity](https://www.audacityteam.org/): it’s free, and it has a db meter and shortcuts for making recording easy.
+
+[Reaper](https://www.reaper.fm/) is also quite good but not yet covered on this document.
```
Comment (LynseyFabel) [inline tts-dataset-recording-at-home.md:18]: please delete if we don't recommend

File: tts-dataset-recording-at-home.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+<u>Do not use software that does not have a numbered db meter.</u>
+
+## How to set up for recording?
```
Comment (LynseyFabel) [inline tts-dataset-recording-at-home.md:22]:
```suggestion
## Recording Prerequisites
```

File: tts-dataset-recording-at-home.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+    
+  
+7.  Set up the boom filter or windscreen.
+    
+<img width="450" alt="image" src="https://github.com/mgrafu/tutorials/assets/47233618/c08c2fbd-0f3d-46e9-aa92-f51519d183d6">
+    
+    
+8.  Select the most quiet room in your environment, and close your windows and doors.
+    
+9.  Eliminate external sources of noise, e. g. air conditioning, computer fan, kids, etc...
```
Comment (LynseyFabel) [inline tts-dataset-recording-at-home.md:55]:
```suggestion
9.  Eliminate external sources of noise, for example, air conditioning, computer fan, and so on.
```

File: tts-dataset-recording-at-home.md [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+
+## How to adjust microphone levels and body position before recording?
```
Comment (LynseyFabel) [inline tts-dataset-recording-at-home.md:57]:
```suggestion
## Adjusting the Microphone Level and Body Position Before Recording
```

File: tts-dataset-recording-at-home.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+    
+<img width="1000" alt="image" src="https://github.com/mgrafu/tutorials/assets/47233618/21a99b40-523a-423b-8cd0-15ffd7dc7f8d">
+    
+    
+    
+- If you’re recording with 16-bit, make sure the db meter is between -12 db and -6 db while you’re recording.
+    
+7.  Do not change the microphone gain after you’ve adjusted it.
+    
+## What happens if you position yourself just right, too far or too close?
```
Comment (LynseyFabel) [inline tts-dataset-recording-at-home.md:87]:
```suggestion
## Positioning Yourself Just Right, Too Far or Too Close
```

File: tts-dataset-recording-at-home.md [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+<img width="1000" alt="image" src="https://github.com/mgrafu/tutorials/assets/47233618/cb082600-7323-4e39-9e3b-4b68a62f29a5">
+    
+
+    
+- **2 feet away** – lots of room sound and bad signal-to-noise ratio.
+    
+<img width="1000" alt="image" src="https://github.com/mgrafu/tutorials/assets/47233618/3851e09e-b62b-4e71-8a3c-dda3ec65892b">
+
+
+## How to record the TTS data?
```
Comment (LynseyFabel) [inline tts-dataset-recording-at-home.md:105]:
```suggestion
## Recording the TTS Data
```

File: tts-dataset-recording-at-home.md [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+
+1.  Prepare your script such that you can easily read it.
+    
+2.  Press Shift+R to record into a new track, read the first sentence, and press space bar to stop recording.
```
Comment (LynseyFabel) [inline tts-dataset-recording-at-home.md:109]:
```suggestion
2.  Press **Shift+R** to record into a new track, read the first sentence, and press space bar to stop recording.
```

File: tts-dataset-recording-at-home.md [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+    
+3.  Repeat Step 2 for each sentence in your dataset until you have completed the recording of the last one.
+    
+4.  Export all files by clicking on File –> Export –> Export Multiple...
```
Comment (LynseyFabel) [inline tts-dataset-recording-at-home.md:113]:
```suggestion
4.  Export all files by clicking on **File** > **Export** > **Export Multiple...**.
```

File: tts-finetune-nemo.ipynb [hunk 14]
```diff
@@ -187,7 +187,7 @@
...
+    "For the rest of this notebook, we will be using a subset of audio samples from the [hifiTTS dataset](https://vctk-subset.s3.amazonaws.com/vctk_subset.tar.gz) adding up to about one minute. This dataset is for demo purposes only. For a good quality model, we recommend at least 30 minutes of audio. If you want to record your own dataset, you can follow the [Guidelines to Record a TTS Dataset at Home](https://github.com/nvidia-riva/tutorials/blob/main/tts-dataset-recording-at-home.md). Sample scripts to download and preprocess datasets supported by NeMo can be found [here](https://github.com/NVIDIA/NeMo/tree/main/scripts/dataset_processing/tts).\n",
```
Comment (LynseyFabel) [inline tts-finetune-nemo.ipynb:190]:
```suggestion
    "For the rest of this notebook, we will be using a subset of audio samples from the [Hi-Fi TTS dataset](https://vctk-subset.s3.amazonaws.com/vctk_subset.tar.gz) adding up to about one minute. This dataset is for demo purposes only. For a good quality model, we recommend at least 30 minutes of audio. If you want to record your own dataset, you can follow the [Guidelines to Record a TTS Dataset at Home](https://github.com/nvidia-riva/tutorials/blob/main/tts-dataset-recording-at-home.md). Sample scripts to download and preprocess datasets supported by NeMo can be found [here](https://github.com/NVIDIA/NeMo/tree/main/scripts/dataset_processing/tts).\n",
```

File: tts-finetune-nemo.ipynb [hunk 15]
```diff
@@ -249,7 +249,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's now download the data from the NVIDIA Custom Voice Recorder tool, and place the data in the `DATA_DIR`. Name the manifest file as `manifest.json`."
+    "Let's now download the hifiTTS audio samples, and place the data in the `DATA_DIR`. Create a manifest file named `manifest.json` and copy the contents of both `dev.json` and `train.json` into it."
```
Comment (LynseyFabel) [inline tts-finetune-nemo.ipynb:252]:
```suggestion
    "Let's now download the Hi-Fi TTS audio samples, and place the data in the `DATA_DIR`. Create a manifest file named `manifest.json` and copy the contents of both `dev.json` and `train.json` into it."
```

File: tts-finetune-nemo.ipynb [hunk 16]
```diff
@@ -260,7 +260,7 @@
    "source": [
     "import os\n",
     "\n",
-    "# Name of the untarred dataset from the NVIDIA Custom Voice Recorder.\n",
+    "# Name of the untarred hifiTTS audio samples directory.\n",
```
Comment (LynseyFabel) [inline tts-finetune-nemo.ipynb:263]:
```suggestion
    "# Name of the untarred Hi-Fi TTS audio samples directory.\n",
```

**Review by LynseyFabel — APPROVED** [review]
I've finished my review and approved this MR assuming the comments are addressed. Thank you.

File: tts-dataset-recording-at-home.md (hunk 6 above)
Comment (mgrafu) [reply tts-dataset-recording-at-home.md:18]: we do recommend this (highly) but don't yet have a full tutorial for it

**Review by virajkarandikar — APPROVED** [review]

**Pull request merged**